### ISSUE TITLE
Help on possible overfit

### ISSUE THREAD

**Issue by processadd** [jdb78/pytorch-forecasting#938]
PyTorch-Forecasting version: current master branch
PyTorch version: 1.9.0
Python version: 3.8

Hi,
I'm using 1 year hourly data to train. Just a single target and time (and another const column as group_id). I use
```
data['time_idx'] = (data['hourly'] - pd.Timestamp("1970-01-01")) // pd.Timedelta("1h")
data["time_idx"] -= data["time_idx"].min()
```

The data is basically 6000~9000, almost no trend, and no seasonality. I use last 14 days to predict next 3 days (viz. 72 hours as it is hourly data). I use lr=3E-4 or 3E-5 and hidden_size=500. After hundreds of epochs, the training loss can be less than 20, but validation loss might be 90~160. If I checked the trained data, it is pretty good and the predict on trained data is like:
<img width="799" alt="image" src="https://user-images.githubusercontent.com/27667124/161682681-ffc49891-feba-4107-9ddc-e2aa0c7f9f0a.png">
mean loss is just 3.7%, but for new data it is like:
<img width="837" alt="image" src="https://user-images.githubusercontent.com/27667124/161682825-a0d390f3-86ce-4221-934f-a79d07ffb78a.png">
or 
<img width="801" alt="image" src="https://user-images.githubusercontent.com/27667124/161682851-a9750817-981f-438d-a32b-36147978f898.png">
the mean loss is around 4~5%, not too bad, but the max loss is 10~13%. More importantly, seems for different days, the predict pattern is very similar. If I use moving window to use 14 days to predict next 3 days, each time the predicted sequence has very similar changing pattern (as the last 2 screenshots). For new data, the predicted values seems missing some 'details'. 

Is there something I can do to make the predict on new behave closer like on trained data?

Thanks
